woocommerce/woocommerce
# Fix some bugs in syncpack configuration

### All Submissions:






### Changes proposed in this Pull Request:

This fixes some bugs in the syncpack configuration and also fixes some packages to specific versions that were not before.

This fixes some build breaking issues I was seeing in a recent PR of mine.






### How to test the changes in this Pull Request:



1. Ensure the build still works: `TURBO_FORCE=true pnpm run build` from project root
2. CI checks passing should confirm everything else needed to test here (lint etc)






### FOR PR REVIEWER ONLY:
